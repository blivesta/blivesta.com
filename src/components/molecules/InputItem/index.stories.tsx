import React from 'react';
import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import Input from '../../atoms/Input';
import InputItem from '.';

const Wrapper = styled.div`
  width: 480px;
  padding: 60px;
`;

storiesOf(`molecules|InputItem`, module)
  .add(`default`, () => (
    <Wrapper>
      <InputItem title="foo" htmlFor="foo">
        <Input id="foo" placeholder="foo" />
      </InputItem>
    </Wrapper>
  ))
  .add(`required`, () => (
    <Wrapper>
      <InputItem title="bar" htmlFor="bar" required>
        <Input id="bar" aria-required required />
      </InputItem>
    </Wrapper>
  ));
