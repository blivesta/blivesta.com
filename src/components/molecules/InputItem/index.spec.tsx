import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import InputItem from '.';

describe('InputItem', () => {
  it('snapshot', () => {
    const tree = shallow(<InputItem title="title">...</InputItem>);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
