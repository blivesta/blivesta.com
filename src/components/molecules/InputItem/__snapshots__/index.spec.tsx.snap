// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputItem snapshot 1`] = `
<P>
  <styled.label>
    title
  </styled.label>
  <styled.div>
    ...
  </styled.div>
</P>
`;
